
A tiny "hack" for our /lib vs. /usr/lib situation. Too damn kde* uses an
own copy of libtool - we need to verify how the official GNU beast behaves
and maybe really fix that one.

  - Rene Rebe <rene@rocklinux.org>

--- kdeadmin-3.1.4/admin/ltmain.sh.orig	2003-12-03 16:30:02.000000000 +0100
+++ kdeadmin-3.1.4/admin/ltmain.sh	2003-12-03 16:30:56.000000000 +0100
@@ -1963,9 +1963,12 @@
 	    dir="$ladir"
 	    absdir="$abs_ladir"
 	    libdir="$abs_ladir"
-	  else
+	  elif test -f "$libdir/$linklib" ; then
 	    dir="$libdir"
 	    absdir="$libdir"
+	  else
+	    dir="/lib"
+	    absdir="/lib"
 	  fi
 	else
 	  dir="$ladir/$objdir"
